docs: add docker compose docs (#145)
codesandbox/docs: packages/projects-docs/pages/tutorial/getting-started-with-docker.mdx

Note that you also have `root` access inside the terminal, that's because we give root access by default inside a container. With this, you can install new packages and use them in your sandbox from the terminal as well. Keep in mind that it's still better to configure this in the `Dockerfile` in the long term, as these temporary container changes won't persist between Dockerfile rebuilds.

![htop running inside the container](/docker-terminal.png)

> ### 4. Create docker compose file

Reviewer: ```suggestion
### 4. Create Docker Compose file
```

> Once the environment is setup with `.codesandbox/Dockerfile`, you can run additional services that are required for the dev environment using the native docker compose integration in CodeSandbox.

Reviewer: ```suggestion
Once the environment is set up with `.codesandbox/Dockerfile`, you can run additional services that are required for the dev environment using the native docker compose integration in CodeSandbox.
```

> Create a new file `docker-compose.yml` in `.codesandbox` folder. As an example you can add the following contents in the file to run an nginx server.

Reviewer: ```suggestion
Create a new file `docker-compose.yml` in the `.codesandbox` folder. For example, you can add the following contents in the file to run an nginx server.
```
